### PR TITLE
Create to-do details view

### PR DESCRIPTION
**Pull request by jmsalcido**
Description:
- change color of the `ic_pin` to the accent attribute
- make the CardView `clickable` to show the cool ripple effect
- open the `DetailActivity` when clicking over the CardView
- rename `ToDoMain` to `MainActivity`

Handles #11

### CODE REVIEW

**Comment by jmsalcido** [comment]
@ ivanebernal please review

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/models/ToDoContent.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package com.nearsoft.androidschool.todoapp.models;
 
-public class ToDoContent {
+import java.io.Serializable;
+
+public class ToDoContent implements Serializable {
```
Comment (CarlosMiguelCuevas) [inline todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/models/ToDoContent.java:5]: shouldn't we use Parcelable? I mean for the purpose of this course is the same, but maybe we could use Parcelable.

File: todoapp/TODOapp/app/src/main/res/layout/list_item.xml [hunk 2]
```diff
@@ -26,6 +28,7 @@
             android:layout_width="0dp"
             android:layout_height="match_parent"
             android:layout_weight="3"
+            android:ellipsize="end"
```
Comment (CarlosMiguelCuevas) [inline todoapp/TODOapp/app/src/main/res/layout/list_item.xml:31]: a very cool property, I didn't know it exist.

File: todoapp/TODOapp/app/src/main/res/layout/activity_todomain.xml [hunk 3]
```diff
@@ -5,21 +5,28 @@
     android:id="@+id/activity_todo_main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:padding="@dimen/activity_vertical_margin"
-    tools:context="com.nearsoft.androidschool.todoapp.activities.main.ToDoMain">
+    tools:context="com.nearsoft.androidschool.todoapp.activities.main.MainActivity">
 
-    <android.support.v7.widget.RecyclerView
-        android:id="@+id/rvTODOList"
+    <RelativeLayout
```
Comment (CarlosMiguelCuevas) [inline todoapp/TODOapp/app/src/main/res/layout/activity_todomain.xml]: do we need to wrap the recycler view into this relative layout? I mean aren't we adding more views to the hierarchy view? or is it a good practice, I might be wrong.

**Review by CarlosMiguelCuevas — COMMENTED** [review]
Just some comments, if they don't apply then LGTM :D

File: todoapp/TODOapp/app/src/main/res/layout/activity_todomain.xml (hunk 3 above)
Comment (jmsalcido) [reply todoapp/TODOapp/app/src/main/res/layout/activity_todomain.xml]: it does not matter, removed.

File: todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/models/ToDoContent.java (hunk 1 above)
Comment (jmsalcido) [reply todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/models/ToDoContent.java:5]: implement it if you want it.

**Pull request merged**